### ISSUE TITLE
Just comes up with a black text box

### ISSUE THREAD

**Issue by Cleancookie** [dciccale/nativeColorPicker#1]
(I know it's an old project but just a heads up)
After clicking a few times this comes up in the console

![console](https://user-images.githubusercontent.com/16965353/27962337-83441c38-6329-11e7-86c6-1d7c342bd7b6.JPG)

**Comment by dciccale** [comment]
I don't see this happening on mac. And I don't have a windows pc at hand. Thanks for the notice though! I will check this at some point hopefully :)

**Comment by chrisbalicki** [comment]
Any joy with this @dciccale? I am getting the same error when running in IE

**Comment by ickata** [comment]
Hi all,

It appears that `<OBJECT CLASSID="clsid:3050f819-98b5-11cf-bb82-00aa00bdce0b"></OBJECT>` does not provide `ChooseColorDlg` interface anymore. Maybe it was removed in IE11, maybe it depends on the platform (Windows 10). But I can't find anything written on this matter, this is just a guess.

**Comment by SavageCore** [comment]
I have one machine it works on and one not, both have the same version of Windows and IE.

## IE:
Version: 11.112.17134.0
Update Versions: 11.0.70 (KB4230450)

## Windows 10:
Version 1803 (OS Build 17134.112)

**Comment by ickata** [comment]
Hi @SavageCore ,

Interesting. Can you please try to list installed plug-ins for IE on both VMs? I have a feeling that the reason for this not working is missing (not installed) plug-in.

**Comment by SavageCore** [comment]
Sure, [working](https://i.imgur.com/Vv3qWv4.png) and [not working](https://i.imgur.com/VQD3vwv.png) add-ons list.

**Comment by ickata** [comment]
Interesting... "Not working" env has all plugins as the "working" one and even more...

**Comment by ickata** [comment]
I just [started a topic in MSDN](https://social.msdn.microsoft.com/Forums/en-US/5a4dc847-3b02-4aa2-b61a-758fac942961/ie-11-does-not-provide-interface-to-access-native-color-picker?forum=iewebdevelopment) about this issue. Let's hope we will get a reasonable explanation.

**Comment by ickata** [comment]
Hi All,

I made this work by doing 2 things:

1. Ensure that ActiveX Filtering is disabled: 
![screen shot 2018-07-12 at 14 59 05](https://user-images.githubusercontent.com/617081/42633585-a448eb46-85e9-11e8-96ab-6c02cc38339d.png)
2. Running IE as Administrator.

[Issue was resolved here](https://answers.microsoft.com/en-us/ie/forum/ie11-iewindows_10/ie-11-does-not-provide-interface-to-access-native/93c1cda9-4c6a-4aec-99c3-a0eff7513167?messageId=77986acb-5cca-4d48-bfbc-6f309b3eba76).

Obviously we can't enforce users to do 1 and 2, therefore using this polyfill for IE is definitely not an option.
